dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/ZenodoHelperTest.java

```diff
@@ -29,4 +29,11 @@ public void testcreateWorkflowTrsUrl() {
         Assert.assertEquals("https://dockstore.org/api/api/ga4gh/v2/tools/%23workflow%2Fgithub.com%2FDataBiosphere"
                 + "%2Ftopmed-workflows%2FUM_variant_caller_wdl/versions/1.32.0/PLAIN-WDL/descriptor/topmed_freeze3_calling.wdl", trsUrl);
     }
+
+    @Test
+    public void extractDoiFromDoiUrl() {
```

Comment:
It would be good to have negative test cases as well (you're only testing the happy path)

Reply:
added negative test extractDoiFromBadDoiUrl